docs/components/password/index.html

```diff
@@ -0,0 +1,147 @@
+            <hx-checkbox-control>
+              <input
+                id="chkHasAsterisk"
+                type="checkbox"
+                v-model="hasAsterisk"
+              />
+              <label for="chkHasAsterisk">
+                <hx-checkbox></hx-checkbox>
+                Required
+              </label>
+            </hx-checkbox-control>
+          </fieldset>
+
+          <fieldset>
+            <legend class="beta-hxFieldName">Control Options</legend>
+            <hx-checkbox-control>
+              <input
+                id="chkDisabled"
+                type="checkbox"
+                v-model="isDisabled"
+              />
+              <label for="chkDisabled">
+                <hx-checkbox></hx-checkbox>
+                Disabled
+              </label>
+              <p>
+                (Invalid styling applied <i>after</i> user interaction.)
+              </p>
+            </hx-checkbox-control>
+          </fieldset>
+
+          <fieldset>
+            <legend class="beta-hxFieldName">Optional Text</legend>
+            <hx-checkbox-control>
+              <input
+                id="chkHasHelpText"
+                type="checkbox"
+                v-model="hasHelpText"
+              />
+              <label for="chkHasHelpText">
+                <hx-checkbox></hx-checkbox>
+                Help Text
+              </label>
+            </hx-checkbox-control>
+            <hx-checkbox-control>
+              <input
+                id="chkHasError"
+                type="checkbox"
+                v-model="hasError"
+              />
+              <label for="chkHasError">
+                <hx-checkbox></hx-checkbox>
+                Error Text
+              </label>
+            </hx-checkbox-control>
+
+            <hx-text-control class="align">
+                <label> Help Text</label>
+              <input type="text" v-model="helpText" :disabled="!helpTextToDisplay">
+             </hx-text-control>
+
+             <hx-text-control class="align">
+                <label> Error Text</label>
+              <input type="text" v-model="errorText" :disabled="!errorTextToDisplay">
+             </hx-text-control>
+          </fieldset>
+        </form>
+      </header>
+
+      <div>
+        <hx-password-control>
+          <input
+            id="pwdInputDemo"
+            type="password"
+            :disabled="isDisabled"
+            :required="isRequired"
+            v-model="label"
+            pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}"
+          />
+          <label
+            for="pwdInputDemo"
+            :class="{ hxOptional: hasOptional, hxRequired: hasAsterisk , hxOptional1: hxOptional2}"
+          >
+          Password
+          </label>
+        </hx-password-control>
+
+        <hx-text-control>
+          <p :class="{hxHelpText: hasHelpText}" v-if="helpTextToDisplay">
+             {% raw %}{{helpText}}{% endraw %}
+         </p>
+        </hx-text-control>
```

Comment:
1. Please remove this `<hx-text-control>`.
1. Move the `<p>` into `<hx-password-control>`.
1. Update the logic to toggle the `hidden` attribute on `<p>`.
1. Keep the `hxHelpText` class, and toggle `hidden`.